src/com/nastia/Arrays/Task1.java

```diff
@@ -2,6 +2,9 @@
 
 public class Task1 {
     public static void main(String[] args) {
-        System.out.println("ll111l");
+        String [] Months = {"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"};
+        for (String i: Months) {
```

Comment:
String [] months = {"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"};
        for (String month : months) {